src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/services/MIReportingServiceTest.java

```diff
@@ -97,6 +99,16 @@ void should_save_task_and_get_task_from_replica_tables() {
 
 
 
+    @Test
```

Comment:
Can we also add a test to prove workTypes table is populated with expected data?